manifest.yaml

```diff
@@ -135,3 +136,12 @@ backup:
     mounts:
       BACKUP: "/mnt/backup"
       main: "/root/data"
+#migrations:
+#  from:
+#    "*":
+#      type: script
+#      args: ["from"]
+#  to:
+#    "*":
+#      type: script
+#      args: ["to"]
```

Comment:
See note above - fine to leave this in and commented out - but add an explanation as to what it is for and why it is commented out, or implement a simple solution :)